test/input.txt

```diff
@@ -154,4 +158,4 @@
 
   # exits properly
   exit
-  list
```

Comment:
Why is this a change when nothing changed? Perhaps a newline or whitespace character?

Reply:
There doesn't appear to be any whitespace or new line. It doesn't seem to affect the test though. Strange..